src/hooks/useSelect/index.js

```diff
@@ -198,6 +198,12 @@ function useSelect(userProps = {}) {
         type: stateChangeTypes.MenuKeyDownEnter,
       })
     },
+    ' '(event) {
```

Comment:
I am wondering if whether it would be better to use `Space` instead of `' '` here?

Reply:
we should be safe here according to https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values

Although I would probably name it `Spacebar` instead of `SpaceButton` in all the occurrences in this PR. For example `stateChangeTypes.MenuKeyDownSpacebar`. What do you think?

Reply:
`Space` wouldn't work for some reason. I guess that's why it's `' '` on another place as well.

https://github.com/downshift-js/downshift/blob/84b6ff6046507a8ea3cd783e005778e97d2a9674/src/downshift.js#L626

I also use `SpaceButton` because I found that name being used else where.

https://github.com/downshift-js/downshift/blob/84b6ff6046507a8ea3cd783e005778e97d2a9674/src/downshift.js#L628

But `Spacebar` does make more sense to me TBH. I could update them all if you think that's okay.

Reply:
It does make more sense. But thinking about it a bit more, in order to be consistent, we need to deliver a breaking change, and change those occurrences as well.

Nobody complained about it so far, let's leave it like this.

Reply:
Can you merge latest master, remove size snapshot, run build locally and update that size snapshot and commit it?

Reply:
Sorry I just read your reply. Seems like you already merge master yourself?